cluster/manifests/aws-fsx-csi-driver/daemonset.yaml

```diff
@@ -0,0 +1,117 @@
+# Source: aws-fsx-csi-driver/templates/node-daemonset.yaml
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: fsx-csi-node
+  namespace: kube-system
+  labels:
+    application: kubernetes
+    component: aws-fsx-csi-driver
+spec:
+  selector:
+    matchLabels:
+      app: fsx-csi-node
+      application: kubernetes
+      component: aws-fsx-csi-driver
```

Comment:
let's use `daemonset: fsx-csi-node` as selector

Reply:
Ah, seems to have missed this. Thanks for catching it! 🙏